crates/svm-cli/tests/app_template_tests.rs

```diff
@@ -55,12 +97,13 @@ fn encode_invalid_codepath() {
     ];
 
     let matches = cli::new_app().get_matches_from(input);
-    match cli::process(matches) {
-        Ok(_) => panic!(),
-        Err(e) => assert!(e
-            .to_string()
-            .starts_with(&format!("failed to open file at {}", code_path))),
-    }
+    let res = cli::process(matches);
+    assert!(res.is_err());
+    assert!(res
+        .err()
+        .unwrap()
+        .to_string()
+        .starts_with(&format!("failed to open file at {}", code_path)));
```

Comment:
please try to assert against a struct/enum that represents the error:
something like: 

```rust
assert_eq!(Err(FileOpenError { ..} ), res);
```

The best error here is probably `std::io::Error` (for `FileNotFound`)
https://doc.rust-lang.org/std/io/struct.Error.html

Reply:
see https://github.com/spacemeshos/svm/pull/109#discussion_r404111736